docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-VERSION       = SNAPSHOT
+VERSION       = 1.4.0
```

Comment:
What's the reason to update the version here?

Reply:
This is a mistake, good catch!

https://github.com/neo4j/neo4j-graphrag-python/pull/286/commits/1c42743630cbda7c43de2fb3dd6bc936e8eb33d4